lib/semantics/resolve-names.cc

```diff
@@ -79,7 +79,7 @@ class ImplicitRules {
 // Provide Post methods to collect attributes into a member variable.
 class AttrsVisitor {
 public:
-  void BeginAttrs();
+  bool BeginAttrs();
```

Comment:
Need a comment explaining when it returns true or false?

Reply:
OK, I'll add one.